VACMS-2509: Remove old blobby content model for Facility health services

netboss opened the pull request:
## Description

closes #2509 
~Blocked by https://github.com/department-of-veterans-affairs/content-build/pull/1031~
Blocked by https://github.com/department-of-veterans-affairs/content-build/pull/1032   This can not be merged until https://github.com/department-of-veterans-affairs/content-build/pull/1032#pullrequestreview-907821757 is merged AND makes its way into CMS codebase.

- Removed "field_body" field from VAMC Facility health service content type.
- Updated behat feature.
- Added hook_update_n in va_gov_vamc module.

## Testing done
Passes all tests

## Screenshots
![VACMS-2509-hook_update_N-returned-message](https://user-images.githubusercontent.com/846871/157829426-4270d64a-8391-4982-969b-2513058db36f.png)

![VACMS-2509-update php-returned-message](https://user-images.githubusercontent.com/846871/157829449-52b249fc-82a3-40f4-8646-fb1f9a3ecb9d.png)

## QA steps
As user administrator
1. Go to `/admin/structure/types/manage/health_care_local_health_service/fields`
   - [x] Verify and confirm the `field_body` (label: **Facility description of service**) is not part of the listed fields
2. Go to VAMC System Banner Alert with Situation Updates: `/admin/structure/types/manage/full_width_banner_alert/fields`
    - [x] Verify and confirm the `field_body` (label: **Alert body**) is listed in the available fields
3. Go to VAMC System Health Service: `/admin/structure/types/manage/regional_health_care_service_des/fields`
    - [x] Verify and confirm the `field_body` (label: **VAMC system service description**) is listed in the available fields
4. Verify and confirm that hook_update_N returned a successful message:
    - [x] In a terminal run `ddev drush updb` and check that this message is printed out: "_The field field_body have been removed from the VAMC Facility Health Service content type._"
    - [x] Run `update.php` (either by going to `/update.php` or run it from the toolbar menu) and check that this message is printed out: "_The field field_body have been removed from the VAMC Facility Health Service content type._"

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [x] `Sitewide CMS Team`
  - [ ] `⭐️ Content ops`
  - [ ] `⭐️ CMS Experience`
  - [ ] `⭐️ Offices`
  - [ ] `⭐️ Product Support`


### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

swirtSJW commented:
This absolutely can not go out until the FE PR is merged and makes its way in to our repo.

netboss commented:
> This absolutely can not go out until the FE PR is merged and makes its way in to our repo.

@swirtSJW 
I didn't find any FE template using the field_body field so there is no FE (content-build) PR for this.

swirtSJW commented on docroot/modules/custom/va_gov_vamc/va_gov_vamc.install:
```diff
@@ -22,3 +23,17 @@ function va_gov_vamc_update_9001() {
+    $field->delete();
+
+    Drupal::logger('va_gov_vamc')->log(LogLevel::INFO, 'VAMC Facility Health Service: Successfully removed field_body field.');
+  }
```
Ideally hook_update_n should return a string message.  It gets output in the terminal record or on screen if using the update.php UI

swirtSJW commented:
Hooray for end to end testing.  :)  content-build failing test is because of this https://github.com/department-of-veterans-affairs/va.gov-cms/issues/2509#issuecomment-1064532903

swirtSJW commented:
One of the QA steps should be to validate the presence of the log message.

netboss commented:
> One of the QA steps should be to validate the presence of the log message.

@swirtSJW 
Done. Updated QA steps and added screen shots of both update via terminal and via browser.

swirtSJW reviewed (dismissed):
Code looks good.  It will need to be-retested after resolving the merge conflict.

swirtSJW commented:
Nice work on improving the QA steps.     Not tests will not pass on this until the change to content-build makes its way into  CMS repo.

netboss commented:
@swirtSJW  
I think this is ready. The content-build PR was merged already. Also, I tested it in tugboat and all worked fine.

swirtSJW reviewed (approved):
All good.  This is no longer blocked and is ready to go.

swirtSJW commented:
![image](https://user-images.githubusercontent.com/5752113/160616209-c423b078-5f0a-48a2-adda-87087a19abfd.png)
